dev-install-locally-no-tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+./gradlew clean install -x test
```

Comment:
This one for instance:
https://github.com/Jasig/cas/blob/master/dev-build-no-tests.sh

You added these, remember? :D

I have no objections to the change, just the organization of it.